### PR TITLE
feat: DAH-2225 Newest Batch of Translations

### PR DESCRIPTION
**Pull request by jimlin-sfgov**
https://sfgovdt.jira.com/browse/DAH-2225

### CODE REVIEW

File: app/assets/json/translations/react/es.json [hunk 1]
```diff
@@ -1222,14 +1231,18 @@
...
+  "lottery.nextStepsLearnMore": "Obtenga más información acerca de qué esperar después del sorteo.",
   "lottery.nextStepsP1": "El agente se comunicará con los solicitantes según el orden de preferencia hasta que se hayan ocupado todas las vacantes.",
   "lottery.nextStepsP2": "Si su solicitud sale elegida en el sorteo, tenga en cuenta que deberá completar una solicitud más detallada y proporcionar la documentación de respaldo requerida.",
   "lottery.nextStepsP3": "Si se ocupan todas las vacantes para una preferencia, usted seguirá siendo elegible para las unidades de su preferencia siguiente.",
   "lottery.nextStepsTitle": "¿Qué sucede después?",
   "lottery.numberApplicantsQualifiedForGeneralPool": "Hay %{number} solicitantes en esta lista. Tenga en cuenta que los titulares de preferencias se considerarán primero para todas las unidades.",
   "lottery.numberApplicantsQualifiedForPreference": "Hay %{number} solicitantes en esta lista.",
+  "lottery.numberApplicantsQualifiedForPreference.nonVeteran": "%{number} otros solicitantes %{plural} en esta lista",
+  "lottery.numberApplicantsQualifiedForPreference.veteran": "%{number} Veteranos%{plural} en esta lista",
```
Comment (chadbrokaw) [inline app/assets/json/translations/react/es.json:1242]: This string does not have a singular translation? This should be "1 veterano en esta lista"
Screenshot 2024-06-17 at 9 07 29 AM

File: app/assets/json/translations/react/es.json [hunk 2]
```diff
@@ -1222,14 +1231,18 @@
...
   "lottery.lotteryResultsPdfLink": "Descargue los Resultados del Sorteo en Formato PDF",
+  "lottery.nextStepsLearnMore": "Obtenga más información acerca de qué esperar después del sorteo.",
   "lottery.nextStepsP1": "El agente se comunicará con los solicitantes según el orden de preferencia hasta que se hayan ocupado todas las vacantes.",
   "lottery.nextStepsP2": "Si su solicitud sale elegida en el sorteo, tenga en cuenta que deberá completar una solicitud más detallada y proporcionar la documentación de respaldo requerida.",
   "lottery.nextStepsP3": "Si se ocupan todas las vacantes para una preferencia, usted seguirá siendo elegible para las unidades de su preferencia siguiente.",
   "lottery.nextStepsTitle": "¿Qué sucede después?",
   "lottery.numberApplicantsQualifiedForGeneralPool": "Hay %{number} solicitantes en esta lista. Tenga en cuenta que los titulares de preferencias se considerarán primero para todas las unidades.",
   "lottery.numberApplicantsQualifiedForPreference": "Hay %{number} solicitantes en esta lista.",
+  "lottery.numberApplicantsQualifiedForPreference.nonVeteran": "%{number} otros solicitantes %{plural} en esta lista",
```
Comment (chadbrokaw) [inline app/assets/json/translations/react/es.json:1241]: This string appears to be still being machine translated
Screenshot 2024-06-17 at 9 09 46 AM

File: app/assets/json/translations/react/es.json [hunk 3]
```diff
@@ -1017,6 +1019,11 @@
   "listings.lotteryPreference.Treasure Island Resident (TIR) Preference.title": "Preferencia de residente de Treasure Island (TIR)",
   "listings.lotteryPreference.allRemainingUnits": "Todas las unidades restantes",
   "listings.lotteryPreference.remainingUnitsAfterPreferenceConsideration": "Después de que se haya tenido en cuenta a todos los titulares preferenciales, las unidades restantes estarán disponibles para los solicitantes calificados en el orden del sorteo.",
+  "listings.lotteryPreference.sfusd.allOtherEmployees.shortTitle": "Todos los demás empleados, SFUSD",
+  "listings.lotteryPreference.sfusd.allOtherEmployees.title": "Todos los demás empleados, Distrito Unificado de Escuelas de San Francisco",
+  "listings.lotteryPreference.sfusd.educators.shortTitle": "Personal docente, SFUSD",
+  "listings.lotteryPreference.sfusd.educators.title": "Personal docente, Distrito Unificado de Escuelas de San Francisco",
+  "listings.lotteryPreference.sfusd.generalPublic.shortTitle": "Público general",
```
Comment (chadbrokaw) [inline app/assets/json/translations/react/es.json:1026]: Going to https://dahlia-webapp-pr-2195.herokuapp.com/listings/a0W4U00000NlolaUAB and entering code 01425245 (following the screenshot on the doc) I don't see this but rather 
Screenshot 2024-06-17 at 9 20 06 AM

Am I doing something wrong?

File: app/assets/json/translations/react/es.json [hunk 4]
```diff
@@ -1213,7 +1220,9 @@
   "lottery.enterLotteryNo": "Ingrese su número de sorteo",
   "lottery.enterLotteryNumber": "Ingrese su número de sorteo",
   "lottery.generalPool": "Pozo general",
+  "lottery.generalPublic": "Público general",
```
Comment (chadbrokaw) [inline app/assets/json/translations/react/es.json:1223]: What code should I use to see this string? Looking at the doc I'm not sure

File: app/assets/json/translations/react/es.json (hunk 3 above)
Comment (jimlin-sfgov) [reply app/assets/json/translations/react/es.json:1026]: From the Jira ticket (and also mentioned in the translation doc):
> Some translations only apply to SCV 2 listings, which do not exist in Full yet.

You would have to create the new SCV 2 type of listing in Salesforce. It turns out that there will be new changes to SCV 2, so I think we can ignore this translation for now. I included it for the sake of completeness.

File: app/assets/json/translations/react/es.json (hunk 4 above)
Comment (jimlin-sfgov) [reply app/assets/json/translations/react/es.json:1223]: Same thing here as the other comment about `generalPublic`

File: app/assets/json/translations/react/es.json (hunk 1 above)
Comment (jimlin-sfgov) [reply app/assets/json/translations/react/es.json:1242]: Our translation system doesn't properly support pluralization. I discussed with Bridget and we decided that any pluralization attempts in this translation batch will inevitably lead to some small translation issues, but we can ignore them for now and fix them once this ticket is completed (which is blocked by a bloom UIC ticket): https://sfgovdt.jira.com/browse/DAH-2276?focusedCommentId=79363

File: app/assets/json/translations/react/es.json (hunk 3 above)
Comment (chadbrokaw) [reply app/assets/json/translations/react/es.json:1026]: Sounds good! Thank you for clarifying!

**Review by chadbrokaw — APPROVED** [review]

**Pull request merged**